MapboxCoreNavigationTests/CocoaPodsTest/PodInstall/Podfile.lock

```diff
@@ -27,7 +27,7 @@ DEPENDENCIES:
   - MapboxNavigation (from `../../../`)
 
 SPEC REPOS:
-  trunk:
+  https://github.com/cocoapods/specs.git:
```

Comment:
This fixes failures in the CocoaPods installation tests that I inadvertently introduced in 8fa1138646090f83901ddd7498b679000dda4587 for #2275, which I overlooked due to expected failures in the lint step.